app/src/lib/stores/app-store.ts

```diff
@@ -441,6 +441,19 @@ export class AppStore extends TypedBaseStore<IAppState> {
     }
   }
 
+  public _resumeTutorial(repository: Repository) {
+    if (this.currentOnboardingTutorialStep === TutorialStep.Paused) {
```

Comment:
Should this be the other way around?

```suggestion
    if (this.currentOnboardingTutorialStep !== TutorialStep.Paused) {
```

Without this change I'm unable to resume the tutorial from the welcome flow.

Reply:
yep, good catch thank you!